### ISSUE TITLE
Loss function

### ISSUE THREAD

**Issue by xsk0802** [Po-Hsun-Su/pytorch-ssim#38]
Hello, I want to use SSIM as a loss function in my own project. How can I reference your code?

**Comment by Aopotai** [comment]
maybe use pip install pytorch_ssim
